src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+interface Props {
+  todo: Todo
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const { user, title, completed } = todo;
+
+  return (
+    <>
+      {user
+        ? <UserInfo user={user} />
+        : null}
```

Comment:
this code returns null or UserInfo
```suggestion
      {user && <UserInfo user={user} />}
```